docs/application-setup.md

```diff
@@ -1,6 +1,6 @@
 # Application setup
 
-Prior to the implementation of Luigi, you need to set up your application. This document shows you how to set up a web application using the Luigi micro front-end framework.
+Prior to start developing with Luigi, you need to set up your application. This document shows you how to set up a web application using the Luigi micro front-end framework.
 
```

Comment:
I think it makes more sense how it is expressed currently, since that makes more clear that Luigi is used as a dependency in your web application. It could be rephrase in another way though, but this is out of scope for this task.

Reply:
Ok - let's leave it this way.